compiler/src/main/scala/org/scalajs/core/compiler/PrepJSExports.scala

```diff
@@ -168,10 +177,11 @@ trait PrepJSExports { this: PrepJSInterop =>
     val exports = directExportsOf(sym) ++ inheritedExportsOf(sym)
 
     // Calculate the distinct exports for this symbol (eliminate double
-    // occurrences of (name, isNamed) pairs).
-    val grouped = exports.groupBy(exp => (exp.jsName, exp.isNamed))
+    // occurrences of (name, isNamed, isTopLevel) tuples).
+    val grouped = exports.groupBy(exp =>
+        (exp.jsName, exp.isNamed, exp.isTopLevel))
```

Comment:
With `()`, the `exp =>` part should be on the same line as the body of the lambda.